guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,317 @@
+
+![Selection9](https://i.imgur.com/aWGEO3P.png)
+
+- Next step run command `ip link show` to see which network interfaces are up  it will look like the picture bellow
+- You should have you loopback `lo` and your network adapters `enp0s3` and `enp0s8` or similars. 
+
+![Selection9](https://i.imgur.com/2r0ghbi.png)
+
+- You will need to add another interface to the server which is going to be the ip address for your lap to go into th einternet. 
+- Run command on the terminal `sudo nano /etc/network/interfaces` and enter your admin password
+
+![Selection9](https://i.imgur.com/EmMewtr.png)
+
+- Then you will see a text file like bellow
+
+![Selection9](https://i.imgur.com/YzAyL1s.png)
+
+- Then add the new network adapter like bellow
+- ISP sever network interface 
+-	`auto enp0s8`
+		`iface enp0s8 inet static`
+			`address 172.31.11.1`
+			`netmask 255.255.255.248`
+			`up route add -net 172.0.0.0 netmask 255.0.0.0 gw 172.31.1.2`
+
+![Selection9](https://i.imgur.com/cU2izxY.png)
+
+- Then press `control X` to exit and `Y` to save file then enter to leave the file with the same name do not change the file name
+
+![Selection9](https://i.imgur.com/gCgEyWU.png)
+
+- run `sudo reboot` on the terminal this will reboot the machine and the seeting on the interfaces will take effect
+- Then run `router` on the terminal after it reboot it shout look like picture bellow 
+
+![Selection9](https://i.imgur.com/wCEIwLd.png)
+
+- Then run the next line on terminal this will add the new network adapter to your network route. 
+- `sudo route add –net 172.0.0.0 netmask 255.0.0.0 gw 172.31.1.3 dev enp0s8`
+
+![Selection9](https://i.imgur.com/AveqQd6.png)
+
+- Then run `route` again to verify it took effect 
+
+![Selection9](https://i.imgur.com/I9QDTpD.png)
+
+- Then run `sudo services networking restart` to restard the network services nothing will display it only going to ask you for your admin password. This will restard your network services for the code above take effect. 
+
+![Selection9](https://i.imgur.com/hR7cdtQ.png)
+
+- Then to verify everything is working fine run `ifconfig` the enp0s8 should have the infomation you enter like bellow
+
+![Selection9](https://i.imgur.com/b4lVh9W.png)
+
+- Before running the following command run `sudo apt-get update and upgrade` and `sudo apt-get install iptables-persistant` this will make the followin lines work correctly.  
+- Then run this tree line on terminal like the picture bellow this will enable persistent on the iptables. 
+- `sudo apt-get install -y iptables-persistent`
+
+![Selection9](https://i.imgur.com/xncEGuM.png)
+
+- Then on the next two question select yes like bellow
+
+![Selection9](https://i.imgur.com/yjjPrcx.png)
+![Selection9](https://i.imgur.com/Fjd9QBr.png)
+
+- Run next line bellow and password if ask it will not display anything after you run the line same as above is to make the iptables not reset.
+- `sudo iptables -t nat -A POSTROUTING -o enp0s3 -j MASQUERADE`
```

Comment:
Explain why you run this command.
Like so:
Now that are iptables rules have the ability to be permanent we will add new rules to ensure our network can work. We will be using `sudo iptables -t nat -A POSTROUTING -o enp0s3 -j MASQUERADE` which creates a [NAT/PAT](link-this-to-a-nat-video-to-explain-it-since-its-too-long-to-explain-here) rule. Now any traffic leaving your CCDC network will leave and be given a public IP so any private IP addresses will work.